exporters/trace/jaeger/src/main/java/io/opencensus/exporter/trace/jaeger/JaegerExporterHandler.java

```diff
@@ -152,27 +159,47 @@ public Tag apply(final Object value) {
 
   private final ThriftSender sender;
   private final Process process;
+  private final Duration deadline;
 
-  JaegerExporterHandler(final ThriftSender sender, final Process process) {
+  JaegerExporterHandler(final ThriftSender sender, final Process process, Duration deadline) {
     this.sender = checkNotNull(sender, "Jaeger sender must NOT be null.");
     this.process = checkNotNull(process, "Process sending traces must NOT be null.");
+    this.deadline = deadline;
   }
 
   @Override
   public void export(final Collection<SpanData> spanDataList) {
     final Scope exportScope = newExportScope();
     try {
```

Comment:
nit: any reason not to use a try-with-resources?

Reply:
Compatibility of this exporter is JDK 1.6: https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/trace/jaeger/build.gradle#L4.